src/Functions/wkt.cpp

```diff
@@ -52,6 +55,7 @@ class FunctionWkt : public IFunction
             for (size_t i = 0; i < input_rows_count; ++i)
             {
                 std::stringstream str; // STYLE_CHECK_ALLOW_STD_STRING_STREAM
+                str.exceptions(std::ios::failbit);
```

Comment:
Here was a bug.

Reply:
There are quite a few other places using stringstream's without setting the exceptions bit (just look for `STYLE_CHECK_ALLOW_STD_STRING_STREAM`. Should all of them set it?

Reply:
Yes, all places should have it. Otherwise, any exception, e.g. bad_alloc, will lead to a silent loss of data.